pkg/compiler/types.go

```diff
@@ -612,6 +613,13 @@ var typeString = &typeDesc{
 		if t.Ident == stringnoz && len(args) > 1 {
 			comp.error(args[0].Pos, "fixed-size string can't be non-zero-terminated")
 		}
+		if t.Ident == glob {
+			if len(args) == 0 {
+				comp.error(t.Pos, "glob only accepts 1 arg, provided %v", len(args))
```

Comment:
It looks like the same message duplicated twice.
I would structure it as we do above for stringnoz check.